php/utils.php

```diff
@@ -1505,3 +1504,72 @@ function format_table( $items ) {
 	}
 	\EE::log( $delem );
 }
+
+/**
+ * Get the site-name from the path from where ee is running if it is a valid site path.
+ *
+ * @return bool|String Name of the site or false in failure.
+ */
+function get_site_name() {
+	$sites = EE::db()::select( array( 'sitename' ) );
+
+	if ( $sites ) {
+		$cwd          = getcwd();
+		$name_in_path = explode( '/', $cwd );
+		$site_name    = array_intersect( array_flatten( $sites ), $name_in_path );
+
+		if ( 1 === count( $site_name ) ) {
```

Comment:
Add an else condition here. In that condition Display the message that informs user like  "There are multiple sites in database matching your path fragments. Please provide the sitename".

Reply:
Review isn't relevant as if condition is being removed.